be/src/exprs/math_functions.cpp

```diff
@@ -851,4 +851,113 @@ template StatusOr<ColumnPtr> MathFunctions::cosine_similarity<TYPE_FLOAT, true>(
 template StatusOr<ColumnPtr> MathFunctions::cosine_similarity<TYPE_FLOAT, false>(FunctionContext* context,
                                                                                  const Columns& columns);
 
+template <LogicalType TYPE, bool isNorm>
```

Comment:
To me, l2 distance doesn't have a notion of norm

Reply:
Agree. @yulongfufu do you have demane on non-norm L2 distance calculation?

Reply:
> To me, l2 distance doesn't have a notion of norm / 
Agree. @yulongfufu do you have demane on non-norm L2 distance calculation?

yes! l2 distance does not have the concept of norm, and it doesn't make any sense. Moreover, we don't have any requirements in this regard. The implementation of the norm method is merely based on a kind of symmetrical aesthetics. I'll remove it.